tabular/src/autogluon/tabular/models/ensemble/greedy_weighted_ensemble_model.py

```diff
@@ -81,3 +81,10 @@ def get_info(self):
         info = super().get_info()
         info['model_weights'] = self._get_model_weights()
         return info
+
+    @classmethod
+    def _get_default_ag_args(cls) -> dict:
+        default_ag_args = super()._get_default_ag_args()
+        extra_ag_args = {'valid_base': False}
```

Comment:
@sxjscience This would be how to specify the stack logic for a model. For BERT, it might make sense for the default to be `{'valid_stacker': False}`.